custom/bihar/management/commands/bihar_cleanup_case.py

```diff
@@ -18,55 +51,51 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         csv_file = csv.writer(open('bihar_case_cleanup.csv', 'wb'))
-        csv_file.writerow(['case_id', 'old_case_type', 'new_case_type',
-                           'old_owner_id', 'new_owner_id', 'parent_id',
-                           'parent_owner_id'])
-
-        blank_case_type_key = ["all type owner", "care-bihar", "", MOTECH_ID]
-        blank_case_ids = [c['id'] for c in
-                CommCareCase.view('case/all_cases',
-                    startkey=blank_case_type_key,
-                    endkey=blank_case_type_key + [{}],
+        csv_file.writerow(CaseRow.headers)
+
+        blank_case_type_keys = [
+            ["all type", "care-bihar", ""],
+            ["all type", "care-bihar", None]
+        ]
+        blank_case_ids = []
+        for key in blank_case_type_keys:
+            blank_case_ids += [
+                c['id'] for c in
+                CommCareCase.view(
+                    'case/all_cases',
+                    startkey=key,
+                    endkey=key + [{}],
                     reduce=False,
-                    include_docs=False
-                ).all()]
-        task_case_ids = [c['id'] for c in
-                CommCareCase.get_all_cases("care-bihar", case_type="task")]
+                    include_docs=False,
+                ).all()
+            ]
 
-        case_ids = set(blank_case_ids) | set(task_case_ids)
+        # task_case_ids = [c['id'] for c in
```

Comment:
Since you already ran this I assumed that was already done but if that's not the case then I can add it back